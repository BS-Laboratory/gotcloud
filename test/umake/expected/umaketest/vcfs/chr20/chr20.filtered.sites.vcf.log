Analysis started on Wed Jan 30 11:24:37 2013

The following filters are in effect:
dp60 : DP >= 60.00
DP1200 : DP <= 1200.00
AB70 : AB <= 0.70
STR20 : STR <= 0.20
str-20 : STR >= -0.20
stz-5 : STZ >= -5.00
STZ5 : STZ <= 5.00
AOI5 : AOI <= 5.00
fic-10 : FIC >= -0.10
INDEL5 : INDEL >= 5 bp with /home/mktrost/gotcloudPlay/test/umake/ref/indels/1kg.pilot_release.merged.indels.sites.hg19.chr20.vcf

Writing to VCF file /home/mktrost/gotcloudPlay/A2/umaketest/vcfs/chr20/chr20.filtered.sites.vcf
Analysis finished on Wed Jan 30 11:24:37 2013

