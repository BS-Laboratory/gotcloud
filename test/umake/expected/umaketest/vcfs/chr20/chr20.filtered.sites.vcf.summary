--------------------------------------------------------------------------------------------------------------
              FILTER	#SNPs	#dbSNP	%dbSNP	%CpG	%CpG	%Known	%Novel	%nCpG-K	%nCpG-N	%HM3	%HM3
                    				Known	Novel	Ts/Tv	Ts/Tv	Ts/Tv	Ts/Tv	sens	/SNP
--------------------------------------------------------------------------------------------------------------
              INDEL5	9	8	88.9	0.0	100.0	NA	0.00	NA	NA	0.003	11.111
          INDEL5;SVM	2	2	100.0	0.0	NA	NA	NA	NA	NA	0.000	0.000
                PASS	1032	1000	96.9	16.0	3.1	2.46	3.00	2.23	2.88	0.571	20.543
                 SVM	74	21	28.4	9.5	1.9	2.00	0.47	1.71	0.49	0.000	0.000
--------------------------------------------------------------------------------------------------------------
              FILTER	#SNPs	#dbSNP	%dbSNP	%CpG	%CpG	%Known	%Novel	%nCpG-K	%nCpG-N	%HM3	%HM3
                    				Known	Novel	Ts/Tv	Ts/Tv	Ts/Tv	Ts/Tv	sens	/SNP
--------------------------------------------------------------------------------------------------------------
              INDEL5	11	10	90.9	0.0	100.0	NA	0.00	NA	NA	0.003	9.091
                PASS	1032	1000	96.9	16.0	3.1	2.46	3.00	2.23	2.88	0.571	20.543
                 SVM	76	23	30.3	8.7	1.9	2.29	0.47	2.00	0.49	0.000	0.000
--------------------------------------------------------------------------------------------------------------
                PASS	1032	1000	96.9	16.0	3.1	2.46	3.00	2.23	2.88	0.571	20.543
                FAIL	85	31	36.5	6.5	3.7	3.43	0.46	3.14	0.49	0.003	1.176
               TOTAL	1117	1031	92.3	15.7	3.5	2.48	0.91	2.25	0.93	0.573	19.069
------------------------------------------------------------------------------------------------------------
