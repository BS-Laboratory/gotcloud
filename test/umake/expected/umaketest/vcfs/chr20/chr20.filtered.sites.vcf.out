gotcloud/bin/invNorm --in vcfs/chr20/chr20.filtered.sites.vcf.raw --out vcfs/chr20/chr20.filtered.sites.vcf.norm

Available Options
                Input options : --in [vcfs/chr20/chr20.filtered.sites.vcf.raw],
                                --ignore
               Output Options : --out [vcfs/chr20/chr20.filtered.sites.vcf.norm],
                                --digits [3], --rank, --verbose
   Output compression Options : --plain [ON], --bgzf, --gzip

invNorm 1.1.1 -- Column-wise inverse-normal transformation of matrix
(c) 2011 Hyun Min Kang and Goo Jun

Reading positive examples from chr20Ref/1000G_omni2.5.b37.sites.chr20.PASS.vcf.gz 
Reading positive examples from chr20Ref/hapmap_3.3.b37.sites.chr20.vcf.gz 
Positive samples: 287, Negative samples: 37
DP	MQ	NS	AN	AB	AZ	FIC	SLRT	LBS	OBS	STR	STZ	CBR	CBZ	IOR	IOZ	AOI	AOZ	LQR	MQ0	MQ10	MQ20	MQ30	QUAL
gotcloud/bin/svm-train -s 0 -t 2 vcfs/chr20/chr20.filtered.sites.vcf.labeled.svm vcfs/chr20/chr20.filtered.sites.vcf.svm.model
*
optimization finished, #iter = 96
nu = 0.072267
obj = -11.833510, rho = 0.260973
nSV = 45, nBSV = 3
Total nSV = 45
gotcloud/bin/svm-predict vcfs/chr20/chr20.filtered.sites.vcf.svm vcfs/chr20/chr20.filtered.sites.vcf.svm.model vcfs/chr20/chr20.filtered.sites.vcf.svm.pred
Accuracy = 0% (0/1117) (classification)
